pcdsdevices/pseudopos.py

```diff
@@ -167,4 +168,23 @@ class SimDelayStage(DelayBase):
 
 class PseudoSingleInterface(PseudoSingle, FltMvInterface):
     """PseudoSingle with FltMvInterface mixed in."""
-    pass
+    notepad_setpoint = Cpt(
+        NotepadLinkedSignal, ':OphydSetpoint',
+        notepad_metadata={'record': 'ao', 'default_value': 0.0},
+    )
+
+    notepad_readback = Cpt(
+        NotepadLinkedSignal, ':OphydReadback',
+        notepad_metadata={'record': 'ai', 'default_value': 0.0},
+    )
+
+    def __init__(self, prefix='', parent=None, **kwargs):
+        if not prefix:
+            # PseudoSingle generally does not get a prefix. Fix that here,
+            # or 'notepad_setpoint' and 'notepad_readback' will have no
+            # prefix.
+            prefix = parent.prefix
```

Comment:
Is there any protection against someone trying to instantiate a `PseudoSingleInterface` on its own, without a parent?

Reply:
Well I guess that should be done in `PseudoSingle`, not here anyways

Reply:
You're absolutely right there should be some checks to ensure it's a component in a pseudopositioner device hierarchy - but I think the risk is pretty low here, as even ophyd-aware people are confused by what these PseudoSingles and such are 😀